pkg/validate/validate.go

```diff
@@ -0,0 +1,49 @@
+package validate
+
+import (
+	"fmt"
+
+	"github.com/mt-sre/addon-metadata-operator/api/v1alpha1"
+)
+
+type Validator struct {
+	Description string
+	Runner      ValidateFunc
+}
+
+type ValidateFunc func(mb *MetaBundle) error
+
+type MetaBundle struct {
+	AddonMeta *v1alpha1.AddonMetadataSpec
+	// TODO: add field for corresponding bundle
+}
+
+// TODO: This will return a MetaBundle with corresponding bundle
+func NewMetaBundle(addonMeta *v1alpha1.AddonMetadataSpec) *MetaBundle {
+	return &MetaBundle{
+		AddonMeta: addonMeta,
+	}
+}
+
+func (mb *MetaBundle) Validate(runMeta bool) []error {
```

Comment:
I think we can drop the `runMeta` option here. We can run all validators globally, and if some of them want to ignore `Bundle` then they can just do so in the func implementation? 

WDYT?